pkg/workload/pgx_helpers.go

```diff
@@ -39,6 +39,9 @@ type MultiConnPoolCfg struct {
 	// If 0, there is no per-pool maximum (other than the total maximum number of
 	// connections which still applies).
 	MaxConnsPerPool int
+
+	// NoPrepare determines whether or not prepared statements can be used or not.
+	NoPrepare bool
```

Comment:
sometimes i find boolean flags that have a negation in the name more confusing. could this just be named `UseSimpleProtocol` or something?